hawkular/metrics.py

```diff
@@ -124,7 +127,13 @@ def _clean_metric_id(metric_id):
         return quote(metric_id, '')
 
     def _get_base_url(self):
-        return "{0}://{1}:{2}/{3}/".format(self.scheme, self.host, str(self.port), self.path)
+        scheme = self.scheme.lower()
```

Comment:
@simon3z checked on my centos, this block is unnecessary, removing it.
I do not know what is the problem on my system, but it is only on my system :-(